qf_lib/data_providers/bloomberg_beap_hapi/bloomberg_beap_hapi_data_provider.py

```diff
@@ -406,11 +430,12 @@ def get_current_values(self, tickers: Union[BloombergTicker, Sequence[BloombergT
         return cast_dataframe_to_proper_type(squeezed_result) if tickers_indices != 0 or fields_indices != 0 \
             else squeezed_result
 
-    def _get_universe_id(self, tickers: Sequence[BloombergTicker], creation_time: Optional[datetime] = None):
+    def _get_universe_id(self, tickers: Sequence[BloombergTicker], creation_time: Optional[datetime] = None,
+                         overrides: Optional = None):
```

Comment:
I would have the type hint for `overrides` as `Optional[List[Tuple]]` since it's a bit unclear from the code what it actually is, I thought it was a boolean at first